cli/pcluster_config/cli.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+#
+
+import argparse
+import logging
+import sys
+
+from pcluster.config.hit_converter import HitConverter
+from pcluster.config.pcluster_config import PclusterConfig, default_config_file_path
+
+
+def _err_and_exit(message):
+    """Log the given error message and exit nonzero."""
+    print(message)
+    sys.exit(1)
+
+
+def _parse_args(argv=None):
+    """Parse command line args."""
+    default_config_file = default_config_file_path()
+
+    parser = argparse.ArgumentParser(
+        description=(
+            "Convert a 'cluster' section from a ParallelCluster's configuration file "
+            "into the Heterogeneous Instance Type format."
```

Comment:
"into a format that supports multiple instance types."

Reply:
Fixed.